traitsui/testing/tester/qt4/default_registry.py

```diff
@@ -21,5 +27,44 @@ def get_default_registry():
         The default registry containing implementations for TraitsUI editors
         that is qt specific.  
     """
+    registry = get_qobject_registry()
+
+    button_editor.register(registry)
+
+
+    return registry
+
+
+def get_qobject_registry():
```

Comment:
I ended up leaving it in the default_registry file, because I thought it might be useful if someone was trying to look into what handlers/solvers are included in the default registry to be able to see them all in one file (at least for the most part aside from specific editor implementations).  The specific handlers (e.g. `mouse_slick_qwidget` are kept in helpers.py).  After thinking again though, maybe it should be in handlers, because realistically a user should never need to think about `QPushButtons` etc., they would only want to know which editors are supported and then they could assume they could do whatever is needed with those editors.  

So, I think I will move it

Reply:
Actually, I may move it after my next PR to avoid conflicts (I have been working on text editor implementation where this function is in the default registry file and don't want to deal with transferring it over and then having to merge).  But I think I will move it eventually.